### PR TITLE
WIP: Add tiebreak option: previous rounds, then config permutation

### PR DESCRIPTION
**Pull request by artoonie**
As a demo for some folks, not intended to merge without discussion

### CODE REVIEW

File: src/main/resources/network/brightspots/rcv/hints_winning_rules.txt [hunk 1]
```diff
@@ -34,6 +34,8 @@ Tiebreak Mode (required): Ties in ranked-choice voting contests can occur when e
 
   * Previous round counts (then random): The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, the tie is broken randomly. Requires a random seed.
```
Comment (bruceleban) [inline src/main/resources/network/brightspots/rcv/hints_winning_rules.txt:35]: Suggest:
> The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, it looks at the round before that. If there are still tied candidates after examining all previous rounds, the tie is broken randomly. Requires a random seed.

This and following suggestions repeat text. Could restructure these to say instead:
> Looks at previous rounds to break ties (see below). If there are still tied candidates after examining previous rounds, the tie is broken randomly. Requires a random seed.

File: src/main/resources/network/brightspots/rcv/hints_winning_rules.txt [hunk 2]
```diff
@@ -34,6 +34,8 @@ Tiebreak Mode (required): Ties in ranked-choice voting contests can occur when e
...
 
+  * Previous round counts (then config permutation): The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, the tie is broken using the order of candidates in the config file.
```
Comment (bruceleban) [inline src/main/resources/network/brightspots/rcv/hints_winning_rules.txt:37]: Suggest:
> The` tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, it looks at the round before that. If there are still tied candidates after examining all previous rounds, the tie is broken using the order of candidates in the config file.

File: src/main/resources/network/brightspots/rcv/hints_winning_rules.txt [hunk 3]
```diff
@@ -34,6 +34,8 @@ Tiebreak Mode (required): Ties in ranked-choice voting contests can occur when e
...
+
   * Previous round counts (then stop counting and ask): The tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, user is prompted to select any tied candidate to eliminate or, in multi-winner contests only, elect.
```
Comment (bruceleban) [inline src/main/resources/network/brightspots/rcv/hints_winning_rules.txt:39]: Suggest:
> The` tied candidate with the least votes in the previous round loses the tie. If there is a tie in the previous round, it looks at the round before that. If there are still tied candidates after examining all previous rounds, the user is prompted to select any tied candidate to eliminate or, in multi-winner contests only, elect.

**Comment by artoonie** [comment]
Closing this, can re-open if discussion with external stakeholders progresses

**Pull request closed without merging**